src/modules/postToAPI.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable class-methods-use-this */
+class LikeCounter {
+  // eslint-disable-next-line camelcase
+  newLike(item_id) {
+    fetch('https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/WXzERyUj1Xacp8cuF1ms/likes', {
+      method: 'POST',
+      body: JSON.stringify({
+        item_id,
+      }),
+      headers: {
+        'Content-type': 'application/json; charset=UTF-8',
+      },
+    })
+      .then((response) => response.text());
+  }
```

Comment:
- [ ] Please kindly stick to using `async and await` for all asynchronous operations